tests/assets/ansible/roles/node_base/tasks/sles-15.yml

```diff
@@ -28,11 +28,22 @@
         - name: "sle_server_updates"
           url: "http://download.suse.de/ibs/SUSE/Updates/SLE-Product-SLES/{{ sle_major }}-SP{{ sle_minor }}/x86_64/update/"
       loop_control:
-        loop_var: repo
+        loop_var: _repo
```

Comment:
Why not format the repository list closer to `tests/assets/ansible/roles/caasp/tasks/Suse.yaml` and then allow the repositories var to be overwitten instead of adding the below `add extra repositories` task?